ctfpad/templates/ctfpad/dashboard/dashboard.html

```diff
@@ -10,6 +10,16 @@
 
 {% include 'snippets/messages.html' %}
 
+<script>
+fetch("{{CODIMD_URL}}/login", {
+  method: 'POST',
+  mode: 'no-cors',
+  cache: 'no-cache',
+  credentials: 'include',
+  headers: { 'Content-Type': 'application/x-www-form-urlencoded'},
+  body: "email={{request.user}}%40localhost.localdomain&password=hedgedoc"});
```

Comment:
Also not a fan of static password: we have access to [`request.user`](https://docs.djangoproject.com/en/3.1/ref/contrib/auth/#user-model) object, that leaves room for crafting unique password per users.

Reply:
yeah i just wanted to have a quick PoC working, feel free to use `request.user.password` or whatever instead. Also I didn't want to bother with what if the user resets or updates their password in ctfpad, then we need to update it in hedgedoc as well etc.